packages/sources/superstate/src/transport/nav.ts

```diff
@@ -78,11 +80,11 @@ export class NavTransport implements Transport<BaseEndpointTypes> {
     return this.execute(fundId, reportValue)
   }
 
-  // Runs 'execute' function every day at 9:09 AM ET (if fundIdsSet is not empty)
+  // Runs 'execute' function every day at START_TIME: 9:08 AM ET (if fundIdsSet is not empty)
   runScheduler() {
     const rule = new schedule.RecurrenceRule()
     rule.hour = 9
-    rule.minute = 9
+    rule.minute = 8
```

Comment:
Can we parse these from START_TIME

Reply:
It would be ideal if we can have those as env vars as well.

Reply:
probably, yeah 😢